kustomization.yaml

```diff
@@ -1,19 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I think we should keep kustomize, it's needed for other integrations we have that install Operator via YAML